### PR TITLE
Update yaml.asciidoc

### PR DESCRIPTION
**Pull request by bsnyder74**
Update deprecated flag.

### CODE REVIEW

**Comment by kvch** [comment]
jenkins, test this, please

File: libbeat/docs/yaml.asciidoc [hunk 1]
```diff
@@ -49,7 +49,7 @@ ifdef::allplatforms[]
 
 ["source","shell",subs="attributes"]
 ----------------------------------------------------------------------
-{beatname_lc} -c {beatname_lc}.yml -configtest
+{beatname_lc} test config -c {beatname_lc}.yml
 ----------------------------------------------------------------------
 
 endif::allplatforms[]
```
Comment (andrewkroh) [inline libbeat/docs/yaml.asciidoc]: The Windows section that follows looks like it needs the same fix.

**Comment by ruflin** [comment]
Just a note that this is against 6.2 and not master. Will this need a forward port or should be opened against master?

**Comment by dedemorton** [comment]
I've cherry-picked your commit into the master branch (see #7360) and will make sure it gets into all relevant branches. Thanks for your contribution!

File: libbeat/docs/yaml.asciidoc (hunk 1 above)
Comment (dedemorton) [reply libbeat/docs/yaml.asciidoc]: I've fixed the Windows section here: #7360

**Pull request closed without merging**